book/src/fees/shielded-fees.md

```
@@ -0,0 +1,231 @@
+
+For `ShieldedTransfer`, the client constructs the bundle so that `total_spent −
+total_output = desired_fee`. The Orchard circuit proves that value is conserved
+(inputs = outputs + value_balance), and the binding signature cryptographically
+commits to the `value_balance`. Mutating `value_balance` after signing will cause
+the binding signature to fail verification.
+
+## The Three-Component Fee Model
+
+The minimum shielded fee has three components:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)